contributing.md (#83)
commonprofiles-care/fhir: CONTRIBUTING.md

# Changes:

>     All changes should have an issue in Github that well describes why and what 
>     the change aims to achieve. All changes should be made infeature branches 

Reviewer: I think allowing the 'why' to be stated in the PR itself is also good, this'll save us from creating issues for small things:

```suggestion
    All changes should have an issue in Github or the PR itself that describes well why and what 
    the change aims to achieve. All changes should be made in feature branches 
```

Reviewer 2: I agree and update the branch accordingly.

>     and shall be code reviewed by at least 2 different companies in the group

Reviewer: ```suggestion
    and shall be code reviewed by at least 2 different companies in the group.
    Small changes must be reviewed quickly (2-3 business days). Larger or more complex
    changes or additions shall be discussed in a regular group meeting before being merged.
```

Reviewer 2: Agree and will implement.

# Release:
    To make a new release, a pull request should be approved by at least 2 
    different companies in the group. 

>     The release should be taged in git with a release number. 

Reviewer: ```suggestion
    The release should be tagged in git with a release number. 
```

Reviewer 2: Agree and will implement.

>     This release number correlate with the version publicized in the IG.

Reviewer 2: "Correlate" should be "correlates"

"version publicized" should be "version published"

Author: Thanks @vjernelov I have now pushed a change
